Library/Homebrew/utils/analytics.sh

```diff
@@ -113,6 +113,8 @@ report-analytics-screenview-command() {
       "${args[@]}" \
       --silent --output /dev/null &>/dev/null & disown
   else
+    echo "$HOMEBREW_CURL https://www.google-analytics.com/debug/collect" \
+      "${args[@]}"
     "$HOMEBREW_CURL" https://www.google-analytics.com/debug/collect \
       "${args[@]}"
```

Comment:
Yes. Sorry that I forgot to post a note here about my commit. (It felt uncontroversial enough that I committed straight away after briefly testing it locally.)